packages/cfpb-layout/src/molecules/heroes.less

```diff
@@ -229,6 +229,7 @@
     }
     .m-hero_subhead {
       .heading-2();
+      font-weight: 500;
```

Comment:
Should this be normal(400) to match the [jumbo hero heading](https://github.com/cfpb/design-system/blob/910baf70fec266f4b9c247d9464dffde87ebc00f/packages/cfpb-layout/src/molecules/heroes.less#L227-L229) weight?

Reply:
Oops, you're correct!